husky_dual_ur5_moveit_config/launch/husky_dual_ur5_moveit_controller_manager.launch

```diff
@@ -0,0 +1,18 @@
+<launch>
+
+  <arg name="real" default="false"/>
+
+  <group if="$(arg real)">
+      <rosparam file="$(find husky_dual_ur5_moveit_config)/config/real_controllers.yaml"/>
+  </group>
+
+  <group unless="$(arg real)">
```

Comment:
@tonybaltovski @paulbovbel 

Simulation controllers are brought up unless the real robot is specified. This is because the real robot uses a different namespace for its controller/command scheme and the yaml files reflect that.